create docker file(modified #1)

comet-syt opened the pull request.

comet-syt committed:
create a docker file

created a docker file and doker-compose file.

docker image is here : https://hub.docker.com/repository/docker/songyutaek/hexa-homepage-2023-frontend

comet-syt committed:
create docker file (modified #1)

1. changed file name (removed 'dev')

2. changed host port number (3000->80)

3. changed the relative path with exact form

4. changed COPY ./ ./ -> COPY . .

5. confirmed using node version 18

SYChoi commented on docker-compose.yml:
```diff
@@ -0,0 +1,12 @@
+      - "80:3000"
+    volumes:
+      - /app/node_modules
+      - ./:/app
```
./:/app은 어느 위치인지... : 이라는 것은 따로 위치 알려주는 녀석이 아닐텐데 ㅎㅎ... 다시 수정 필요할듯

SYChoi commented on docker-compose.yml:
```diff
@@ -0,0 +1,12 @@
+    volumes:
+      - /app/node_modules
+      - ./:/app
+    stdin_open: true
```
stdin_open은 왜 열어놨는지? 이유가 있다면 이 코멘트에 답글로 달아주면 좋을듯

SYChoi commented on Dockerfile:
```diff
@@ -0,0 +1,11 @@
+
+COPY . .
+
+CMD ["npm","run","start"]
```
진작 여기서 npm run build는 어디에...? 그리고 npm run start는 그냥 개발 환경에서 돌리는 명령어라서 실제로 운영에서 돌리는 것은 명령어가 다름. 확인해서 수정해보기 바람.

comet-syt committed:
create docker file(modified #2)

1. installed 'serve' package
2. added build commend in docker file
3. deleted docker-compose file
4. modified README.md file (commends to build & run docker image)

SYChoi commented:
새로운 리퀘스트로 넘어감
